posts/2024-01-31-Performance-Doris_Amoakohene/index.qmd

```diff
@@ -6,9 +6,10 @@ output: html_document
 ---
 
 
-Since August 2023, I have been working on performance testing, which could be useful for expanding the open-source Ecosystem around data.table package in R,This could increase confidence in code contributions by ensuring the sustained efficiency of the data.table package. Consequently, it becomes essential to enhance the package's performance and deliver reliable results to its users.
+Since August 2023, I have been working on performance testing, which could be useful for expanding the open-source Ecosystem around data.table package in R.
+This could increase confidence in code contributions by ensuring the sustained efficiency of the data.table package. Consequently, it becomes essential to enhance the package's performance and deliver reliable results to its users.
 
-It is important that we prevent significant Performance Regression from reaching the data.table Package. Slowness, late execution or big memory usage can be frustrating, any Performance Regression that makes it into a version release will degrade user experience. In this blog post, I will demonstrate the use of benchmarking techniques to verify whether  reported issues on data.table have been successfully resolved.
+It is important that we prevent significant performance regression from reaching the data.table package. Slowness, late execution or big memory usage can be frustrating, any performance regression that makes it into a version release will degrade user experience. In this blog post, I will demonstrate the use of benchmarking techniques to verify whether reported issues on data.table have been successfully resolved.
```

Comment:
what is the difference between slowness and late execution? please clarify. if none, then please delete late execution.

Reply:
done, deleted late execution